src/typeahead/typeahead-window.ts

```diff
@@ -37,8 +45,17 @@ export class NgbTypeaheadWindow {
    * A function used to format a given result before display. This function should return a formated string without any
    * HTML markup.
    */
-  @Input() formatter = (result) => { return `${result}`; };
+  @Input() formatter = toStringFormatter;
```

Comment:
just curious, why replace this?
maybe put it to utils, as we're using it elswhere?